_data/navigation.yml

```diff
@@ -0,0 +1,29 @@
+- text: Project
+  submenu:
+    - text: Why HospitalRun?
+      url: /blog/why-hospitalrun/
+    - text: Features
+      url: /features
+    - text: Try it
+      url: /demo
+    - text: Hospital Stories
+      url: /stories
+    - text: FAQs
+      url: /faq
+    - text: Roadmap
+      url: /roadmap
+    - text: Team
+      url: /team
+- text: Blog
+  url: /blog
+- text: Docs
+  url: https://github.com/HospitalRun/hospitalrun-docs
+  submenu:
+    - text: Users
+      url: https://github.com/HospitalRun/hospitalrun-docs
+    - text: Developers
+      url: https://github.com/HospitalRun/hospitalrun-docs
+- text: Contribute
```

Comment:
This has to be changes to "Community", also in other places

Reply:
Not in this release